src/Components/pages/Categories.js

```diff
@@ -0,0 +1,10 @@
+import Category from '../Book/Category';
+
+const Categories = () => (
+  <div>
+    <Category />
+    <button type="button">Check status</button>
+  </div>
```

Comment:
- [ ] According to requirements, the categories should display only a button with text "Check status", so please remove the `<Category />` you have on line 5 🙏